prep-javascript-operators-and-expressions/prep-javascript-operators-and-expressions-quiz-notes.md

```diff
@@ -0,0 +1,38 @@
+# prep-javascript-operators-and-expressions-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What data type is returned by an arithmetic operation?
```

Comment:
dont forget the quiz questions!

Reply:
@Zhiwutian Hey Brett I added the quiz answers, Is it okay to merge now?